src/main/scala/com/fulcrumgenomics/cmdline/FgBioTool.scala

```diff
@@ -25,10 +25,39 @@ package com.fulcrumgenomics.cmdline
 
 import com.fulcrumgenomics.cmdline.FgBioMain.FailureException
 import com.fulcrumgenomics.sopt.cmdline.ValidationException
+import com.fulcrumgenomics.util.Metric
+import htsjdk.samtools.{SAMFileHeader, SAMProgramRecord}
+
+import scala.annotation.tailrec
+
+/** Stores meta information about the command line use to invoke a tool. */
+case class FgBioToolInfo(name: String, commandLine: String, description: String, version: String) extends Metric {
```

Comment:
I'd be ok with having both the full command line (with default args included) and short command line (without default args included).  Right now, I am providing the former only.

Reply:
I think it would be nice, and relatively easy, to do both.  In fact I would suggest we have:

1. `args: Seq[String]` which stores the actual set of args passed to Main
2. `def commandLineWithoutDefaults: String = args.mkString(" ")`
3. `val commandLineWithDefaults` which is what you have as `commandLine` now.